lib/jekyll/filters.rb

```diff
@@ -323,6 +325,23 @@ def sort_input(input, property, order)
         .map!(&:last)
     end
 
+    # `where` filter helper
+    def compare_property_vs_target(property, target)
+      case target
+      when NilClass
+        # won't catch `""`
+        return true if Array(property).empty?
+      when Liquid::Expression::MethodLiteral # `empty` or `blank`
```

Comment:
Are `empty` and `blank` the only values that would cause this `case` statement to match?

Reply:
I'm not entirely certain but seems so from [this code](https://github.com/Shopify/liquid/blob/f2f467bdbc248e7bf26b4f5e552cfbce6052f811/lib/liquid/expression.rb#L20-L21):
I don't see any other special keywords documented on their site or wiki.

Reply:
Yup. That looks fine after looking at the liquid code now. :thumbsup: